specs/IFrameMemoryUsage.md

```diff
@@ -0,0 +1,306 @@
+IFrame Memory Usage API
+===
+
+# Background
+The WebView2 team has been asked to provide support for monitoring iframe 
+memory usage. The end developer needs to know how much memory each iframe 
+consumes. To do that, they need to get the renderer process ID for each 
+iframe. However, WebView2 only tracks the render frame host for top-level iframes. 

```

Comment:
Is a "render frame host" the same as a "renderer process"? If so, please use consistent terminology to avoid reader confusion. If it's not the same thing, please explain the difference.

Also, does "WebView2 only tracks the render frame host for top-level iframes" imply that the information is available under the old API? If so, what is that old API? Are you referring to CoreWebView2Environment.GetProcessInfos()? That seems to cover all renderer processes, not just the ones for top-level iframes.

Reply:
Yes please use public API terminology.